nautobot/extras/admin.py

```diff
@@ -18,232 +13,6 @@ def order_content_types(field):
     field.choices = [(ct.pk, "{} > {}".format(ct.app_label, ct.name)) for ct in queryset]
 
 
-#
-# Custom fields
-#
-
-
-class CustomFieldForm(forms.ModelForm):
-    class Meta:
-        model = CustomField
-        exclude = []
-        widgets = {
-            "default": forms.TextInput(),
-            "validation_regex": forms.Textarea(
-                attrs={
-                    "cols": 80,
-                    "rows": 3,
-                }
-            ),
-        }
-
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-        order_content_types(self.fields["content_types"])
-
-
-class CustomFieldChoiceAdmin(admin.TabularInline):
-    """
-    Defines the inline formset factory that handles choices for selection type custom fields.
-    The `extra` defines the default number of inline rows that appear in the UI.
-    """
-
-    model = CustomFieldChoice
-    extra = 5
-
-
-@admin.register(CustomField)
-class CustomFieldAdmin(admin.ModelAdmin):
```

Comment:
We probably shouldn't remove CustomFieldAdmin until we have a fix in place for #891.